cognite/neat/rules/_importer/_base.py

```diff
@@ -0,0 +1,32 @@
+import getpass
+from abc import ABC, abstractmethod
+from datetime import datetime
+
+from cognite.neat.rules.models._rules import DomainRules, InformationRules
+
+
+class BaseImporter(ABC):
+    """
+    BaseImporter class which all importers inherit from.
+    """
+
+    @abstractmethod
+    def __init__(self):
+        ...
+
+    @abstractmethod
+    def to_rules(self, *args, **kwargs) -> DomainRules | InformationRules:
```

Comment:
Will we ever have an importer that produces domain rules? That seems like a very limited format for rules.

It it possible have more structure here? Can we avoid `args` and `kwargs`?

Reply:
Good point, I doubt. They will probably either create information or cdf architect rules